src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -800,7 +801,7 @@ private List<LaunchCommand> GetInitializeCommands()
                     // TODO: The last clause for LLDB may need to be changed when we support LLDB on Linux as LLDB's tty redirection doesn't work.
                     if (localLaunchOptions != null &&
                         ((PlatformUtilities.IsWindows() && localLaunchOptions.UseExternalConsole)
```

Comment:
Should we move the `&& localLaunchOptions.UseExternalConsole` to the first line since it is true on all OSs?